pkg/sql/sem/builtins/builtins.go

```diff
@@ -8972,9 +8975,9 @@ WHERE object_id = table_descriptor_id
 				{Name: "start_time", Typ: types.Decimal},
 				{Name: "end_time", Typ: types.Decimal},
 			},
-			ReturnType: tree.FixedReturnType(types.Void),
+			ReturnType: tree.FixedReturnType(types.Int),
```

Comment:
we should think about this more later, but I wonder if we could simplify the args of this builtin to be: full backup stmt, start_time, end time. 

That way, we don't have to manually reason about all the different backup options like encryption_opts, incremental backup uris, (we'll need to support execution locality as well). We could simply reuse the backup stmt parsing code. 

To do this correctly, we just have to ensure that we have the full backup stmt when we eventually call the built in, i believe in the schedule exector.